src/jit/target.h

```diff
@@ -788,8 +788,16 @@ typedef unsigned short          regPairNoSmall; // arm: need 12 bits
   #define RBM_CALLEE_TRASH_NOGC   RBM_CALLEE_TRASH
 
   #define RBM_ALLINT              (RBM_INT_CALLEE_SAVED | RBM_INT_CALLEE_TRASH)
+
+  // RBM_LOWINT is used to set LsraLimitSmallIntSet only.
+  // It is intended for limiting the number of the INT registers 
+  // for stress mode testing. It is important to have a good ratio 
+  // of callee/caller saved registers in the mix to get a good stress testing,
+  // so this ratio should be maintained.
+
 #ifdef UNIX_AMD64_ABI
-  #define RBM_LOWINT              (RBM_EAX|RBM_RDI|RBM_RSI|RBM_EDX|RBM_ECX|RBM_EBX|RBM_ETW_FRAMED_EBP)
+  // On System V the RDI and RSI are not callee saved. Use R12 ans R13 as callee saved registers.
+  #define RBM_LOWINT              (RBM_EAX|RBM_ECX|RBM_EBX|RBM_ETW_FRAMED_EBP|RBM_R12|RBM_R13)
```

Comment:
Since this used only by LSRA and an LSRA only setting, like Carol, i too would prefer it is directly set to LsraLimitSmallIntSet in lsra.h.  We can clean up RBM_LOWINT.